lib/wallet/src/Cardano/Wallet.hs

```diff
@@ -1963,7 +1957,7 @@ buildAndSignTransactionPure
        , Bounded (Index (AddressIndexDerivationType k) (AddressCredential k))
        , IsOwned s k 'CredFromKeyK
```

Comment:
```suggestion
       , IsOwned s 'CredFromKeyK
```

The `IsOwned` class can probably also be changed to remove the `k` parameter. (But perhaps in another pull request).

Reply:
this is a mess, already tried

Reply:
I have a DIFFERENT plan there ...